src/DashboardHub/Bundle/AppBundle/Tests/Spec/DashboardHub/Bundle/AppBundle/Service/DashboardServiceSpec.php

```diff
@@ -392,4 +393,23 @@ function it_should_find_all_by_is_public_and_popular(
 
         $this->findAllByIsPublicAndPopular();
     }
+
+    function it_should_search(
+        EntityManager $em,
+        SecurityContext $securityContext,
+        DashboardRepository $dashboardRepository,
+        Search $search
+    )
+    {
+        $dashboardRepository->search($search)
```

Comment:
Looks ok, but the rest of the code base doesn't have that. So now it is inconsistent - we can reformat as we go along.